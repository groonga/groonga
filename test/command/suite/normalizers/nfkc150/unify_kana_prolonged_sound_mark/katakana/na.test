normalize \
  'NormalizerNFKC150("unify_kana_prolonged_sound_mark", true, \
                     "report_source_offset", true)' \
  "ナーニーヌーネーノー" \
  WITH_CHECKS|WITH_TYPES
