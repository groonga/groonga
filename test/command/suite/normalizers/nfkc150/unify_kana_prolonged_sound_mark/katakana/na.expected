normalize   'NormalizerNFKC150("unify_kana_prolonged_sound_mark", true,                      "report_source_offset", true)'   "ナーニーヌーネーノー"   WITH_CHECKS|WITH_TYPES
[
  [
    0,
    0.0,
    0.0
  ],
  {
    "normalized": "ナアニイヌウネエノオ",
    "types": [
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "katakana"
    ],
    "checks": [
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0
    ],
    "offsets": [
      0,
      3,
      6,
      9,
      12,
      15,
      18,
      21,
      24,
      27
    ]
  }
]
